css/style.css

```diff
@@ -15,5 +14,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.topnav {
+  background-color: rgb(255, 255, 255);
+  overflow: hidden;
+  margin-right: 60px;
+}
+
+/* nav bar */
+.topnav a {
+  float: right;
+  color: #5a5a5a;
+  text-align: center;
+  padding: 1vw 2vw;
+  text-decoration: none;
+  font-size: 2vw;
+  background-color: white;
+  margin-top: 2vw;
+}
+
+/* the color of links on hover */
+.topnav a:hover {
+  color: orangered;
+}
+
+/* Adding bold font to the active/current link */
+.topnav a.active {
+  font-weight: bold;
+}
+.logo {
+  margin-left: 4vw;
+  margin-top: 2vw;
+  height: 3vw;
+  width: 2vw;
+}
+/* styling background image & texts on the image */
+
+.big-image {
+  width: 97%;
+  margin: 0;
+}
```

Comment:
On the image that we are recreating the hero image goes all the way across the screen, a keen eye for detail is really important to make sure that we are recreating it exactly to what the image shows